src/test/java/org/jenkinsci/plugins/github/status/sources/ManuallyEnteredRepositorySourceTest.java

```diff
@@ -33,12 +32,11 @@
 
     @Test
     public void nullName() {
-        ManuallyEnteredRepositorySource instance = Mockito.spy(new ManuallyEnteredRepositorySource("https://github.com/jenkinsci/jenkins"));
-        doReturn(null).when(instance).createName(Matchers.anyString());
+        ManuallyEnteredRepositorySource instance = spy(new ManuallyEnteredRepositorySource("a"));
```

Comment:
some weird mockito issue that only occurred when running all tests via CLI, couldn't reproduce in IDE or `mvn test`.

the test is checking nullName, so faking an impossible case didn't make sense anymore so I removed the mocking and the test works